vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -113,25 +113,25 @@ func NewVolumeDriver(targetURL string, projectID string, hostID string, mountDir
 // map hasn't been initialized, return 1 prevent detach and remove.
 
 // Return the number of references for the given volume
-func (d *VolumeDriver) getRefCount(vol string) uint {
+func (d *VolumeDriver) getRefCount(vol string) (uint, error) {
```

Comment:
it is ambiguous but keeps the mount/unmount logic consistent. I think there are positive and negative sides to both the options. We can agree on one. 
My preference is to have the small fix for this blocker to distinguish between states for Remove command.
@govint @pdhamdhere @shaominchen